sources/packages/api/src/route-controllers/user/user.controller.ts

```diff
@@ -94,4 +100,15 @@ export class UserController extends BaseController {
       accounts,
     };
   }
+  @Get("locations")
+  async getAllUserLocations(
+    @UserToken() userToken: IUserToken,
+  ): Promise<UserLocationDto[]> {
+    const userInSABC = await this.service.getUser(userToken.userName);
```

Comment:
I believe that this call is not needed since the method on line 109 could use the "userToken.userName". Can you check it, please?

Reply:
You are right . Done